templates/institutional/institution-landing-en.html

```diff
@@ -0,0 +1,187 @@
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+		</div>
+	</section>
+</div>
+
+<div class="well well-sm provisional well-bold brdr-0 mrgn-bttm-0">
+	<div class="container">
+		<section>
+			<h2 class="h3 mrgn-tp-md">Contact us</h2>
+			<ul class="wb-eqht list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-3">
+				<li><a href="#">Contact [Institution]</a></li>
+				<li><a href="#">[Top contact task 2]</a></li>
+				<li><a href="#">[Top contact task 3]</a></li>
+			</ul>
+		</section>
+		<div class="row">
+			<section class="col-md-8 pull-left mrgn-bttm-lg">
+				<h2 class="h3 mrgn-tp-md">About the [Institution]</h2>
+				<ul class="list-bld list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-2">
+					<li><a href="#">Mandate</a></li>
+					<li><a href="#">Organizational structure</a></li>
+					<li><a href="#">Transparency</a></li>
+					<li><a href="#">Job opportunities</a></li>
+					<li><a href="#">Reports</a></li>
+					<li><a href="#">Compliance</a></li>
+					<li><a href="#">Enforcement notifications</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li>More: <a href="#">About the [Institution]</a></li>
+				</ul>
+			</section>
+			<div class="col-xs-12 col-md-4 col-sm-5 pull-right">
+				<section class="lnkbx mrgn-bttm-lg">
+					<h2 class="h3">Minister</h2>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+				</section>
+				{%- include components/gc-follow-us.html -%}
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="container">
+	<div class="row">
+		<section class="provisional wb-feeds gc-feeds limit-3 col-md-4">
+			<h2 class="h3">News</h2>
+			<ul class="feeds-cont list-unstyled lst-spcd">
+				<li>
+					<a data-ajax="../../components/feeds/demo/example-en.atom.json" href="#">
+					[Institution] news
+					</a>
+				</li>
+			</ul>
+			<p>More: <a href="#">[Institution] news</a></p>
+		</section>
+		<section class="col-md-8 gc-features">
+			<h2 class="h3">Features</h2>
+			<div class="row wb-eqht">
+				<div class="col-sm-6">
+					<div class="well well-sm brdr-rds-0 eqht-trgt">
+						<img class="img-responsive full-width" src="{{assets}}/img/360x203.png" alt="" />
```

Comment:
Paths to images are broken again, it renders **/img/360x203.png** but should simply render **img/360x203.png**
```suggestion
						<img class="img-responsive full-width" src="img/360x203.png" alt="" />
```

Reply:
(Same problem in French.)